Doc/using/windows.rst

```diff
@@ -762,9 +764,15 @@ on Windows which you hope will be useful on Unix, you should use one of the
 shebang lines starting with ``/usr``.
 
 Any of the above virtual commands can be suffixed with an explicit version
-(either just the major version, or the major and minor version) - for example
-``/usr/bin/python2.7`` - which will cause that specific version to be located
-and used.
+(either just the major version, or the major and minor version).
+Furthermore the 32bit version can be enforced by adding "-32" after the
```

Comment:
"Request[ed]", not "enforce[d]". And we generally spell them "32-bit" and "64-bit" in prose, rather than omitting the hyphen.

Reply:
These come up a few times in the text below, but I'm not commenting on each indivdual one.